DQMOffline/RecoB/interface/JetTagPlotter.h

```diff
@@ -80,6 +84,12 @@ class JetTagPlotter : public BaseBTagPlotter {
 
   // reconstructed jet phi
   FlavourHistograms<double> * dJetRecPhi;
+  
+  // jet Phi larger than requested discrimnator cut
+  FlavourHistograms<double> * dJetPhiDiscrCut;
```

Comment:
I did try but I ended up having to change quite a lot of things, so I did not take it further: we'd like this to be merged soon, to be able to monitor these additional plots. If you don't mind I'll make a separate PR later moving everything to smart pointers.